src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -645,19 +645,25 @@ export default class AircraftCommander {
         let taxiDestination = data[0];
```

Comment:
An you add an empty line above this please?

Reply:
done